aiohttp/multipart.py

```diff
@@ -907,16 +901,14 @@ def write(self, chunk):
                     return
 
         if self._encoding == 'base64':
-            self._encoding_buffer.extend(chunk)
-
-            if self._encoding_buffer:
-                buffer = self._encoding_buffer
-                div, mod = divmod(len(buffer), 3)
-                enc_chunk, self._encoding_buffer = (
-                    buffer[:div * 3], buffer[div * 3:])
-                if enc_chunk:
-                    enc_chunk = base64.b64encode(enc_chunk)
-                    yield from self._writer.write(enc_chunk)
+            if chunk:
+                enc_chunk = base64.encodebytes(chunk)
```

Comment:
Usually incomplete code (I'm not alking about tests/linting/docs) should not reach PR, since it will just uselessly bother reviewers.

Your changes are welcome, but please show us complete working code even if you are unusure about the idea. When the idea is approved, you can add tests/docs and so on.